lib/auth/auth_with_roles.go

```diff
@@ -1540,6 +1540,30 @@ func (a *ServerWithRoles) GetNode(ctx context.Context, namespace, name string) (
 	return node, nil
 }
 
+func (a *ServerWithRoles) checkUnifiedAccess(resource types.ResourceWithLabels, checker resourceAccessChecker, actionVerbs []string, filter services.MatchResourceFilter) (bool, error) {
+	var err error
+	err = a.action(apidefaults.Namespace, resource.GetKind(), actionVerbs...)
```

Comment:
would there be any performance issues because it seems we are checking rbac repeatedly for the same resource kind? (can we do it once per resource kind?) i would get a backend person to review this just in case 😅

Reply:
cc: @rosstimothy

Reply:
You know, I don't even think _this_ is a good solution either. This causes my backend to get 20+ INFO RBAC Access not allowed to list type of messages. 

What I'm curious about is why this isn't checked in `CanAccess` as it seems like it should be there. Perhaps I'm missing context